Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -972,7 +981,10 @@ class CombinatorialKalmanFilter {
         typeFlags.set(TrackStateFlag::HoleFlag);
       }
 
-      trackStateProxy.data().ifiltered = trackStateProxy.data().ipredicted;
+      // @FIXME: MTJ direct index access
+      // trackStateProxy.data().ifiltered = trackStateProxy.data().ipredicted;
+      using PM = TrackStatePropMask;
```

Comment:
Worth making this alias for two uses?

Reply:
I guess not. I've used this alias pattern in a number of places, but it's not critical. I can get rid of it (only here?).

Reply:
Actually, if that is a common alias, maybe it could be aliased once for whole class scope. In any case, this is detail.

Reply:
I removed the use of the alias here.